tubular/tests/test_wordpress_api.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+import mock
+
+from tubular.wordpress_api import WordPressAPI
+
+
+class WordPressAPITests(TestCase):
+
+  def setUp(self):
+    self.base_url = 'https://example.com'
+    self.username = 'tubularworker'
```

Comment:
`tubularworker` should be in CONSTANTS/Settings instead of hard coding it here.

Reply:
Its just a test. We can set any username as we are mocking the API result.